tests/DotNetLightning.Core.Tests/TransactionBolt3TestVectorTests.fs

```diff
@@ -26,7 +26,7 @@ let log =
 let dataPath1 = Path.Join(AppDomain.CurrentDomain.BaseDirectory, "../../..", ("Data/bolt3-tx.json"))
 let data1 = dataPath1 |> File.ReadAllText |> JsonDocument.Parse
 
-let localPerCommitmentPoint = PubKey("025f7117a78150fe2ef97db7cfc83bd57b2e2c0d0dd25eaf467a4a1c2a45ce1486")
+let localPerCommitmentPoint = PerCommitmentPoint <| PubKey("025f7117a78150fe2ef97db7cfc83bd57b2e2c0d0dd25eaf467a4a1c2a45ce1486")
```

Comment:
the commit message of this commit looks like it's fixing something, but you're actually changing the tests; how can this be? does the channel pass before and after this commit?

Reply:
It does. But using a `PerCommitmentPoint` to represent a per commitment point is more correct. And it will be necessary for later commits which call methods on the `PerCommitmentPoint` directly rather than just converting it back to a `PubKey` to use it.